lib/jormungandr/src/Cardano/Wallet/Jormungandr/Compatibility.hs

```diff
@@ -228,6 +228,7 @@ genConfigFile stateDir (BaseUrl _ host port path) = object
             [ "messages" .= String "low"
             , "blocks" .= String "normal"
             ]
+        , "public_address" .= String "ta1svy0mwwm7mdwcuj308aapjw6ra4c3e6cygd0f333nvtjzxg8ahdvxlswdf0"
```

Comment:
With that I get:

```
Error in the overall configuration of the node
Error while parsing the node configuration file: p2p.public_address: invalid value: string "ta1svy0mwwm7mdwcuj308aapjw6ra4c3e6cygd0f333nvtjzxg8ahdvxlswdf0", expected Multiaddr (example: /ip4/192.168.0.1/tcp/443) at line 1 column 162
```
I think the value needs to be something like `/ip4/192.168.0.1/tcp/443` after all...

Reply:
Hmm.. Yeah, we are still looking for an address in the sense of an IP address :sweat_smile:

Reply:
I corrected this

Reply:
but locally see it stucks at 
```
Ports CLI (explicit) tests
[iohk.cardano-wallet.serve:Notice:ThreadId 9801] [2019-08-05 15:25:19.15 UTC] 10 migrations were applied to the database.
```

Reply:
we have already set 8090 port (defaultPort)

Reply:
Hmmm. You provided the same value as the default one used by the wallet server 😶

What about making the config generator take an extra port param, and find an adequate port in the launch command handler (by querying the system for a random available one)?